libbeat/docs/shared-template-load.asciidoc

```diff
@@ -85,6 +85,8 @@ output.elasticsearch.index: "customname-%{[{beat_version_key}]}-%{+yyyy.MM.dd}"
 setup.template.name: "customname"
 setup.template.pattern: "customname-*"
 -----
+WARNING: If {ref}/index-lifecycle-management.html[Index Lifecycle Management] is enabled (which is the default), `setup.template.name` and `setup.template.pattern` are ignored.
```

Comment:
Will this take users to `index-lifecycle-management.html`? From what I can tell from the online docs, the page URL is `ilm.html`: https://www.elastic.co/guide/en/beats/filebeat/master/ilm.html.

Also, should we make the same clarification over here as well? https://www.elastic.co/guide/en/beats/filebeat/master/ilm.html#setup-ilm-option

Reply:
TBH, I just fixed the link provided by the original contributor (didn't really add my own changes).  I think you're right that we need to send users to the Beats docs, so I'll fix that!